src/tools/pontoon/translate.md

```diff
@@ -62,89 +54,70 @@ There are two types of quality check failures: errors and warnings.
+In the `REVIEW TRANSLATIONS` section, the user can approve suggestions or reject suggestions for all selected strings. Upon clicking `APPROVE ALL` or `REMOVE ALL SUGGESTIONS`, the label of the button will be replaced with the number of affected translations.
+Note that `APPROVE ALL` accepts the latest suggestion, but doesn’t reject other suggestions if available.
 
-In the `REPLACE IN TRANSLATIONS` section, the translator can input a series of characters to search for, and a series of characters to use as a replacement. This is a basic find and replace feature that will work only on the selected strings.
+In the `REPLACE IN TRANSLATIONS` section, the user can input the text to search for, and the text to use as a replacement. This is a basic find and replace feature that will work only on the selected strings.
 
-Note that mass actions are only available to users with [translators rights](users.md#user-roles).
+Note that mass actions are only available to users with [translator rights](users.md#user-roles).
 
 ## Downloading and uploading translations
 
-Pontoon features the ability to download and upload translations, including terminology and translation memories. To access these features, click on your user icon in the top-right section of Pontoon. Note that you must be in the translation workspace for the download/upload options to display in the dropdown menu.
+Pontoon provides the ability to download and upload translations, including terminology and translation memories. To access these features, click on the user icon in the top-right section of Pontoon. Note that the user must be in the translation workspace for the download/upload options to be displayed in the dropdown menu.
 
 ![Profile dropdown menu](../../assets/images/pontoon/translate/profile_menu.png)
 
-Anyone can download a terminology (.tbx), translation memory (.tmx) and translations, while only translators can upload translations. When downloading translations:
+Anyone can download terminology (`.tbx`), translation memory (`.tmx`) and translations, while only translators can upload translations. When downloading translations:
 * The resource currently selected is downloaded in its original format, it’s not converted in any way.
-* If the project contains multiple files, a ZIP of all files is downloaded. If the project contains more than 10 files, download is performed file-by-file.
+* If the project contains multiple files, a ZIP of all files is downloaded. If the project contains more than 10 files, only the file currently translated will be downloaded.
 
-When you upload translations to Pontoon:
+When uploading translations to Pontoon:
 * Existing strings with matching translations will be ignored.
 * New strings and new translations will be imported and attributed to the user uploading the file.
```

Comment:
Translations different from the ones in Pontoon will be imported and attributed to the user uploading the file.